recipes/pyphysim/meta.yaml

```diff
@@ -0,0 +1,65 @@
+{% set name = "pyphysim" %}
+{% set version = "0.7.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 0df90c4d8031503d69d2cbd13717daf34cab8295a73aaecdb70e08441cdeaaae
+
+build:
+  noarch: python
+  number: 0
+  script: {{ PYTHON }} -m pip install . -vv
+  missing_dso_whitelist:
+    - "*/libpthread.so.0"  # [linux]
+    - "*/libc.so.6"  # [linux]
```

Comment:
This sounds like the package contains compiled code, that should be rather compiled here and probably makes it non-noarch.